modules/product/product.service.js

```diff
@@ -56,11 +56,10 @@ class ProductsService {
       };
     }
     if (price && price.length) {
-      const currencySign =
-      currency === 0 ? 'UAH' : currency === 1 ? 'USD' : '';
+      const currencySign =      currency === 0 ? 'UAH' : currency === 1 ? 'USD' : '';
```

Comment:
Please, validate currency values via `joi-validator`

Reply:
sry, not mine component